coffe/fpga.py

```diff
@@ -5633,7 +5646,7 @@ def update_area(self):
             self.area_dict["cc_area_total"] = cc_area_total
             self.width_dict["cc_area_total"] = math.sqrt(cc_area_total)
 
-            self.area_dict["local_mux_total"] = local_mux_area + local_mux_area
+            self.area_dict["local_mux_total"] = local_mux_area + local_mux_sram_area
             self.width_dict["local_mux_total"] = math.sqrt(local_mux_area + local_mux_area)
```

Comment:
This line also needs to be changed to  "local_mux_area + local_mux_sram_area". I missed this. I've checked it into my repo. Will submit a pull request for this along with other feedback.